source/MaterialXCore/Interface.cpp

```diff
@@ -73,7 +74,15 @@ bool PortElement::validate(string* message) const
     NodePtr connectedNode = getConnectedNode();
     if (hasNodeName())
     {
-        validateRequire(connectedNode != nullptr, res, message, "Invalid port connection");
+        if (connectedNode == nullptr)
```

Comment:
minor: Don't think I've seen any MaterialX examples where they compare against nullptr, I think they would just write !connectedNode in this case.